packages/components/Pagination.tsx

```diff
@@ -95,7 +97,9 @@ export const Pagination = ({
           name="items"
           label=""
           variant="labelOutside"
-          value={itemsPerPage.toString()}
+          onChangeText={(s) => setItemsPerPage(parseInt(s, 10))}
+          onPressEnter={() => setItemsPerPage(parseInt(itemsPerPage, 10))}
+          value={itemsPerPage}
```

Comment:
When the input is empty, the displayed value is "NaN"

Reply:
changed to be a dropdown